lib/fabrik/DirectorInstance.js

```diff
@@ -249,12 +288,21 @@ class DirectorInstance extends BaseInstance {
       .then(() => this.manager.deleteBinding(this.deploymentName, params.binding_id));
   }
 
-  buildSecurityGroupRules() {
+  buildIpRules() {
+
+    function getPorts(first, last) {
```

Comment:
unnecessary method.

Reply:
will remove as setting [1024, 65535] in ports.